config.sample.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
this needs login_customer_ids as a property as well

Reply:
Added login_customer_ids